### PR TITLE
Cronjobs for nightly subscription payments

### PR DESCRIPTION
**Pull request by roosnic1**
This PR introduces jobs. Jobs can be run in the api=>server context. Jobs have access to the database, paymentProviders, mailProviders etc. In the `examples/api` i added yargs to run the jobs from command line. Actual we.publish instances would run this with some sort of cron management.

⚠️  This PR will merge into `f/WEP-23-member-plans`

### CODE REVIEW

File: examples/api/src/index.ts [hunk 1]
```diff
@@ -224,7 +227,77 @@ async function asyncMain() {
...
+          userPaymentURL: `${websiteURL}/user/invocies`,
+          replyToAddress: 'info@tsri.ch',
+          sendEveryDays: 3
+        })
+        process.exit(0)
+      }
+    )
+    .command(
+      'dic',
+      'charge open invoices',
```
Comment (bishoymelek) [inline examples/api/src/index.ts:292]: By this, You mean to try processing the rejected or newly opened invoices.

**Review by bishoymelek — APPROVED** [review]
:tada: great work and important feature :+1:

File: packages/api/src/jobs.ts [hunk 2]
```diff
@@ -0,0 +1,387 @@
+import {Context} from './context'
+import {DateFilterComparison, InputCursor, LimitType, SortOrder} from './db/common'
+import {UserSort} from './db/user'
+import {PaymentPeriodicity} from './db/memberPlan'
+import {InvoiceSort} from './db/invoice'
+import {PaymentState} from './db/payment'
+
+const ONE_HOUR_IN_MILLISECONDS = 60 * 60 * 1000
```
Comment (penk85) [inline packages/api/src/jobs.ts:8]: Why not just have the result here as opposed to the calculation?

**Review by penk85 — APPROVED** [review]

**Review by faebebin — APPROVED** [review]
Looks good.

File: packages/editor/src/client/panel/paymentMethodEditPanel.tsx [hunk 3]
```diff
@@ -178,7 +175,14 @@ export function PaymentMethodEditPanel({id, onClose, onSave}: PaymentMethodEditP
             </FormGroup>
             <FormGroup>
               <ControlLabel>{t('paymentMethodList.description')}</ControlLabel>
-              <RichTextBlock value={description} onChange={value => setDescription(value)} />
+              <FormControl
+                name={t('paymentMethodList.description')}
```
Comment (0815pascal) [inline packages/editor/src/client/panel/paymentMethodEditPanel.tsx:179]: I guess you forgot to add this to the translation-file?

**Review by 0815pascal — APPROVED** [review]
It looks good, but translation-files are missing. Do you want to add them later on?

**Pull request merged**